File: src/main/scala/resonantinduction/compatibility/nei/RIGrinderRecipeHandler.java
package resonantinduction.compatibility.nei;

import resonant.lib.utility.LanguageUtility;
import resonant.content.factory.resources.RecipeType;

public class RIGrinderRecipeHandler extends RITemplateRecipeHandler
{

	@Override
	public String getRecipeName()
	{
		return LanguageUtility.getLocal("resonantinduction.machine.grinder");
	}

	@Override
	public RecipeType getMachine()
	{
		return RecipeType.GRINDER;
	}
}

File: src/main/scala/resonantinduction/compatibility/nei/RIGrinderRecipeHandler.scala
package resonantinduction.compatibility.nei

import resonant.lib.utility.LanguageUtility
import resonant.content.factory.resources.RecipeType

class RIGrinderRecipeHandler extends RITemplateRecipeHandler
{
    def getRecipeName: String =
    {
        return LanguageUtility.getLocal("resonantinduction.machine.grinder")
    }

    def getMachine: RecipeType =
    {
        return RecipeType.GRINDER
    }
}